### PR TITLE
Add stimulus

### PR DESCRIPTION
**Pull request by thibaultdalban**
install `webpacker` and `stimulus`

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint no-console:0 */
```
Comment (michaelbaudino) [inline app/javascript/packs/application.js]: I don't think we need this, do we?

File: config/webpacker.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+development:
+  <<: *default
+  compile: true
+
+  # Reference: https://webpack.js.org/configuration/dev-server/
+  dev_server:
+    https: false
+    host: localhost
+    port: 3035
+    public: localhost:3035
```
Comment (michaelbaudino) [inline config/webpacker.yml]: Doesn't this require a Docker Compose configuration?

**Comment by michaelbaudino** [comment]
There are conflicts to resolve :crossed_swords:

**Comment by thibaultdalban** [comment]
Rebased

**Pull request merged**